### PR TITLE
Ikasan 2318 attempting to upgrade to springboot 3 manually 4.0.x v6

### PR DESCRIPTION
**Pull request by mick-stewart73**

### CODE REVIEW

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/main/resources/application.properties [hunk 1]
```diff
@@ -17,7 +17,7 @@ h2.db.port=8082
 server.port=8080
 server.address=localhost
 server.servlet.context-path=/${artifactId}
-server.tomcat.additional-tld-skip-patterns=xercesImpl.jar,xml-apis.jar,serializer.jar
+server.tomcat.additional-tld-skip-patterns=*.jar
```
Comment (andrzej-majewski) [inline ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/main/resources/application.properties]: usage of *jar can cause issues with loading blue consel

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/pom.xml [hunk 2]
```diff
@@ -53,15 +53,19 @@
...
-        <maven.compiler.source>17</maven.compiler.source>
-        <version.org.springboot>2.3.4.RELEASE</version.org.springboot>
-        <maven.compiler.target>17</maven.compiler.target>
-        <version.com.h2database>1.4.200</version.com.h2database>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <version.ikasan>3.2.2</version.ikasan>
-        <version.activemq>5.16.3</version.activemq>
     </properties>
 
+<!--    <properties>-->
```
Comment (andrzej-majewski) [inline ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/pom.xml]: why all properties are commented

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/test/resources/application.properties [hunk 3]
```diff
@@ -9,7 +9,7 @@ server.port=8080
 server.address=localhost
 server.contextPath=/sample-db-jms-im
 
-server.tomcat.additional-tld-skip-patterns=xercesImpl.jar,xml-apis.jar,serializer.jar
+server.tomcat.additional-tld-skip-patterns=*.jar
```
Comment (andrzej-majewski) [inline ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/test/resources/application.properties]: can cause issue with blue console

File: ikasaneip/platform/ikasan-setup/src/main/resources/db-changelog-2.1.0.xml [hunk 4]
```diff
@@ -25,7 +25,6 @@
     <include file="db/index/db-changeLog-createIndexFlowModuleId.xml" relativeToChangelogFile="true" />
     <include file="db/index/db-changeLog-createIndexIkasanModuleServerId.xml" relativeToChangelogFile="true" />
     <include file="db/index/db-changeLog-createIndexMetricEventExpiry.xml" relativeToChangelogFile="true" />
-    <include file="db/index/db-changeLog-createIndexFTTransactionalResourceCommandXid_Id.xml" relativeToChangelogFile="true" />
     <include file="db/index/db-changeLog-createIndexMetricEventEventId.xml" relativeToChangelogFile="true" />
```
Comment (andrzej-majewski) [inline ikasaneip/platform/ikasan-setup/src/main/resources/db-changelog-2.1.0.xml:28]: How come this is removed?

File: ikasaneip/transaction/spring/pom.xml [hunk 5]
```diff
@@ -55,10 +55,11 @@
...
 
     <dependencies>
-   
+
         <dependency>
-            <groupId>geronimo-spec</groupId>
-            <artifactId>geronimo-spec-j2ee</artifactId>
+            <groupId>jakarta.transaction</groupId>
+            <artifactId>jakarta.transaction-api</artifactId>
+            <version>2.0.0</version>
```
Comment (andrzej-majewski) [inline ikasaneip/transaction/spring/pom.xml]: version should be upstreamed to main pom

File: ikasaneip/platform/ikasan-setup/src/main/resources/db-changelog-2.1.0.xml (hunk 4 above)
Comment (mick-stewart73) [reply ikasaneip/platform/ikasan-setup/src/main/resources/db-changelog-2.1.0.xml:28]: this somehow made it into the wrong context. This is only relevant for modules that require the ftp and sftp tables.

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/test/resources/application.properties (hunk 3 above)
Comment (mick-stewart73) [reply ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/test/resources/application.properties]: fixed

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/main/resources/application.properties (hunk 1 above)
Comment (mick-stewart73) [reply ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/src/main/resources/archetype-resources/jar/src/main/resources/application.properties]: fixed

File: ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/pom.xml (hunk 2 above)
Comment (mick-stewart73) [reply ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/pom.xml]: those properties are not relevant at that level and in fact require the developer to maintain them. All are derived from parent level poms.

**Pull request merged**